python/GafferSceneUI/RenderUI.py

```diff
@@ -48,7 +48,7 @@ def rendererPresetNames( plug = None ) :
 	return IECore.StringVectorData(
 		sorted(
 			t for t in GafferScene.Private.IECoreScenePreview.Renderer.types()
-			if t not in blacklist
+			if t not in blacklist and Gaffer.Metadata.value( f"renderer:{t}", "enabled" )
```

Comment:
Is it worth having a fallback to `True` for the `enabled` metadata, since hiding seems to be the exception (and on the offchance that someone has an extension for their own renderer)?

I also wonder if `ui:enabled` might be a little clearer in communicating intent?